Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -599,7 +599,7 @@
             'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeInnerZminus.xml',
             'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeOuterZplus.xml',
             'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeOuterZminus.xml',
-            'Geometry/TrackerCommonData/data/PhaseI/v2/pixbarmaterial.xml', 
+            'Geometry/TrackerCommonData/data/PhaseI/v3/pixbarmaterial.xml', 
```

Comment:
Just trying to understand how this backport was designed: this line is from #34983, and it was apparently forgotten in some previous backport of that PR merged in master: could you please explain the logic and the history of previous backports?

Reply:
We had a few PR for 12_1_X and these are combined in the backport PR. Please merge this PR. It is in the way of other back-outings